theano/sandbox/cuda/cuda_ndarray.cu

```diff
@@ -1030,7 +1030,6 @@ CudaNdarray_TakeFrom(CudaNdarray * self, PyObject *args){
         PyObject * indices_float32 = NULL;
         indices_float32 = PyArray_View((PyArrayObject*)indices_obj,
                                                   float32_descr, NULL);
-        Py_DECREF(float32_descr);
```

Comment:
line 1068 is:

Py_DECREF(indices_obj);

but should be:

Py_DECREF(indices);

But I would be surprised this is the problem.